cli/src/lib.rs

```diff
@@ -27,6 +27,7 @@ use chain_spec::ChainSpec;
 use futures::{
 	Future, FutureExt, TryFutureExt, future::select, channel::oneshot, compat::Future01CompatExt,
 };
+#[cfg(not(target_os = "unknown"))]
```

Comment:
In Substrate, this code is in a `cli` module that is behind a `cli` feature:
https://github.com/paritytech/substrate/blob/6e0a4877afbe2309e106b84333bb70200531b604/bin/node/cli/src/lib.rs#L38

What do you think of doing the same here?

Reply:
Done